src/inspector_agent.cc

```diff
@@ -718,6 +874,19 @@ void Agent::InitInspector(Local<Object> target, Local<Value> unused,
   env->SetMethod(target, "connect", ConnectJSBindingsSession);
   env->SetMethod(target, "open", Open);
   env->SetMethod(target, "url", Url);
+
+  env->SetMethod(target, "asyncTaskScheduled", AsyncTaskScheduledWrapper);
+  env->SetMethod(target, "asyncTaskCanceled", AsyncTaskCanceledWrapper);
+  env->SetMethod(target, "asyncTaskStarted", AsyncTaskStartedWrapper);
+  env->SetMethod(target, "asyncTaskFinished", AsyncTaskFinishedWrapper);
+  env->SetMethod(target, "registerAsyncHook", RegisterAsyncHookWrapper);
+
+  auto enabled_string = FIXED_ONE_BYTE_STRING(env->isolate(), "enabled");
+  CHECK(target->SetAccessor(env->context(),
```

Comment:
> use a function call if the value could change in the future

I think I see your point now. I have converted `inspector.enabled` into `inspector.enabled()` function in a5b49c0.